jupyter-book/conditions/differential_gene_expression_environment.yml

```diff
@@ -0,0 +1,20 @@
+name: deg
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.8
```

Comment:
Can we go 3.9 here? We have this for like almost any environment.

Reply:
yes

Reply:
updated to 3.9